ci-operator/jobs/infra-periodics.yaml

```diff
@@ -1195,3 +1195,57 @@ periodics:
     - name: token
       secret:
         secretName: github-credentials-openshift-bot
+- agent: kubernetes
+  cluster: app.ci
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  hidden: true
+  extra_refs:
+  - base_ref: master
+    org: openshift
+    repo: release
+  interval: 12h
+  labels:
+    ci.openshift.io/role: infra
+  name: periodic-ci-secret-bootstrap
+  spec:
+    containers:
+    - args:
+      - --bw-password-path=/etc/ci-secret-bootstrap/bw_password
+      - --bw-user=tbd
```

Comment:
username is not a secret, right?

Reply:
Are we using a user's login or a read-only token?

Reply:
Yes. I got it from BW.
I will update.